content/docs/dvclive/live/log_artifact.md

```diff
@@ -36,21 +37,27 @@ with Live() as live:
 
 Uses `dvc add` to track `path` with DVC, generating a `{path}.dvc` file.
 
-When combined with `save_dvc_exp=True`, it will ensure that `{path}.dvc` is
-included in the experiment.
+When combined with [`save_dvc_exp=True`](/doc/dvclive#initialize-dvclive), it
+will ensure that `{path}.dvc` is included in the experiment.
 
 If `Live` was initialized with `dvcyaml=True` (which is the default), it will
 add an [artifact](/doc/user-guide/project-structure/dvcyaml-files#artifacts) and
 all the metadata passed as arguments to the corresponding `dvc.yaml`. Passing
 `type="model"` will mark it as a `model` for DVC and will make it appear in
 [Studio Model Registry](/doc/studio) (coming soon).
 
-If `name` is not provided, the path stem (last part of the path without the file
-extension) will be used as the artifact name.
-
 ## Parameters
 
-- `path` - existing directory or file
+- `path` - an existing directory or file.
+
+- `name` - an optional custom name of an artifact. If not provided the path stem
+  (last part of the path without the file extension) will be used as the
+  artifact name.
+
+- `copy` - copy a directory or file at `path` into the `dvclive/artifacts`
+  location ([default](/doc/dvclive/how-it-works#directory-structure)) before
+  tracking it. The new path is used instead of the original one to track the
+  artifact.
```

Comment:
@aguschin Sorry I missed this before, but shouldn't we include all the args here? 

Also, do we actually want to support `meta`? Is it used anywhere?

Reply:
We had a GTO user asking for that field previously. No harm in keeping it I assume?

> Is it used anywhere?

We don't use it. The only way to get it currently is call DVC API or read `dvc.yaml`. In GTO there was a command `gto describe myartifact --custom` which would output content of `custom` field (`meta` is the same field for the same purposes, but with a different name).

Reply:
> Sorry I missed this before, but shouldn't we include all the args here?

Right, thanks for the notice.